topics/dev/tutorials/tool-annotation/tutorial.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: tutorial_hands_on
+title: Annotation of a tool with EDAM ontology terms using bio.tools
+level: Introductory
+subtopic: tooldev
+questions:
+- How are Galaxy tools linked to EDAM ontology?
+- How to connect Galaxy tools to bio.tools?
+objectives:
+- Identify Galaxy tools without bio.tools entry
+- Create a bio.tools entry
+- Update a bio.tools entry 
+- Add EDAM ontology terms to a bio.tools entry
+- Link a Galaxy tool to its corresponding bio.tools entry
+time_estimation: 1H
+key_points:
+- Galaxy tools can get EDAM ontology terms from bio.tools
+- bio.tools entry can be created and modified to provide the best EDAM annotations
+- bio.tools entry can easily be added to a Galaxy tool
+contributions:
+  authorship:
+    - bebatut
+
+---
+
+Galaxy offers thousands of tools. A vast majority of these tools come with poor metadata. 
+
+This prevents filtering for all tools in a specific research community or domain, and makes it all but impossible to employ advanced filtering with ontology terms like ones from EDAM or to group tools given ontology to improve the Galaxy tool panel.
```

Comment:
It can also make it difficult to understand the breadth and depth of software options available in Galaxy, particularly if you are a new user.